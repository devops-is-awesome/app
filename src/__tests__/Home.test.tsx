import { expect, it, describe, assert } from 'vitest';

describe('very complicated, much tests', () => {
  it('foo', () => {
    expect(1 + 1).toEqual(2);
    expect(true).to.be.true;
  });

  it('bar', () => {
    assert.equal(Math.sqrt(4), 2);
  });

  it('very wow', () => {
    expect({ foo: 'bar' }).toMatchSnapshot();
  });

  it('neo is real', () => {
    expect(1).toBe(1);
  });
});
