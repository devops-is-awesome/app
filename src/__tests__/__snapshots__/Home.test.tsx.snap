// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`suite name > snapshot 1`] = `
{
  "foo": "bar",
}
`;

exports[`very complicated tests > very wow 1`] = `
{
  "foo": "bar",
}
`;

exports[`very complicated, much tests > very wow 1`] = `
{
  "foo": "bar",
}
`;
